### ISSUE TITLE
UI is broken after update

### ISSUE THREAD

**Issue by ArgoHA** [microsoft/vscode#127568]
## Environment data

-   VS Code version: 1.57.1
-   Jupyter Extension version (available under the Extensions sidebar): v2021.6.999662501
-   Python Extension version (available under the Extensions sidebar): v2021.6.944021595
-   OS (Windows | Mac | Linux distro) and version: Mac OS Big Sur
-   Python and/or Anaconda version: python3.7.3
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A
-   Jupyter server running: Local | Remote | N/A: Local or Remote

I am getting multiple issues after notebook has updated.
- First of all, edges of cells are barely visible with Atom theme (they were visible before). The view is really strange and seems like I am getting less cells in my screen...
- Numbers on Y and X axes are not visible
- The most annoying thing which literally is breaking for me - a solid 0.5 second delay before starting scroll. Scroll feels laggy and irresponsive now.
- Existed shortcuts were lost. My shift + enter doesn't work anymore

<img width="1109" alt="Снимок экрана 2021-06-29 в 21 28 50" src="https://user-images.githubusercontent.com/43640256/123851809-50a56a00-d924-11eb-8855-e3a3757e6a76.png">

Is there's a way to downgrade? It is not usable by now. Really waiting for fixes, this UI might be pretty nice after resolving issues.
P.S. Hope cmd+z is fixed it this version

**Comment by ArgoHA** [comment]
<img width="768" alt="Снимок экрана 2021-06-30 в 07 33 34" src="https://user-images.githubusercontent.com/43640256/123902464-963e5300-d975-11eb-879c-46e119a12cb8.png">
I just put my cursor on first variable and got this strange thing... Seems like a lot of UI bugs, or I don't understand something...

**Comment by joyceerhl** [comment]
The last comment is a dupe of https://github.com/microsoft/vscode-jupyter/issues/5932

**Comment by IanMatthewHuff** [comment]
@ArgoHA This is the new notebook UI that is going to be rolling out to all customers over the next few months. Sorry about some of the pain moving forward, but this new UI is now natively provided by VS Code so it's going to end up being a lot more performance and consistent with VS Code versus the old UI implementation. 

> Existed shortcuts were lost. My shift + enter doesn't work anymore

Per this comment in the new UI each notebook cell is now a fully fledged VS Code document instance. Is it possible that you have a keybinding for shift+enter that might be overriding the notebook shift enter command?

**Comment by ArgoHA** [comment]
Would love it to be with a lot more performance and consistent. Don't understand why I'm having delay at the start of the scroll though. Is my 15inch macbook from late 2016 too old?

Speaking of shift+enter maybe, here is what I've got. Could you show me correct way to use shift + enter to run the cell and go to next one?
<img width="1093" alt="Снимок экрана 2021-06-30 в 20 33 20" src="https://user-images.githubusercontent.com/43640256/124006659-32eb0a00-d9e3-11eb-8573-5bbca5c21280.png">


And I also have some questions about customizing the view. Where can I find docs on that?
I would like to make border a little bit bolder or smth
And I would love to get rid of `+ Code +Markdown` buttons under the cell and make padding between cells a little bit smaller

**Comment by joyceerhl** [comment]
@ArgoHA  You can customize color tokens in your user settings.json like so:

![image](https://user-images.githubusercontent.com/30305945/123904566-88f97800-d925-11eb-8da5-53c2d8bdc100.png)

**Comment by joyceerhl** [comment]
You can find docs on notebook layout settings in the release notes: https://code.visualstudio.com/updates/v1_57#_notebook-layout-customization

**Comment by claudiaregio** [comment]
@ArgoHA To remove the +Code +Markdown from under the cells, click the overflow menu in your notebook toolbar and select Customize Notebook Layout. 

![image](https://user-images.githubusercontent.com/19276747/124159644-3ec0e580-da50-11eb-91dd-b8248fbd1600.png)

Then you'll want to change the setting below to your preferred version. You likely have the value "both" on as that is the default. 

![image](https://user-images.githubusercontent.com/19276747/124159820-6adc6680-da50-11eb-80ac-4c4c4c94cd48.png)

**Comment by joyceerhl** [comment]
> Existed shortcuts were lost. My shift + enter doesn't work anymore

Can you share what your shift+enter keybinding used to do?

**Comment by ArgoHA** [comment]
> Can you share what your shift+enter keybinding used to do?

Run the cell and jump to next one. cmd+enter now runs the cell but focus stays on it.

**Comment by ArgoHA** [comment]
> Then you'll want to change the setting below to your preferred version. You likely have the value "both" on as that is the default.

Thanks, I have it hidden now. So is it possible to make paddings between cells smaller?

**Comment by joyceerhl** [comment]
Just to clarify, shift+enter is now opening a terminal instead of running the current cell and advancing?

If so that sounds like a bug in the Python extension

**Comment by ArgoHA** [comment]
> Just to clarify, shift+enter is now opening a terminal instead of running the current cell and advancing?

Yes, it does


> Do you get the right behavior then?

Yes, I did actually get the right behaviour. What should I do nest?

**Comment by joyceerhl** [comment]
Had you customized keybindings for those commands before? If not, we should open an issue in https://github.com/microsoft/vscode-python to review the when clauses for those keybindings.

**Comment by ArgoHA** [comment]
> Had you customized keybindings for those commands before?

Cant really remember. If I'm not mistaken, shift+enter was not default "run cell and advance" for VS code. It is for jupyter notebook, so when I transitioned to VS code I changed that
Will try to reset key shortcuts now

**Comment by joyceerhl** [comment]
You might try `"when": "editorTextFocus && !notebookEditorFocused && !notebookCellListFocused"`

**Comment by claudiaregio** [comment]
> Thanks, I have it hidden now. So is it possible to make paddings between cells smaller?

@ArgoHA There is something else you could do here -- go back to the notebook layout settings and click the pencil to the right to edit the setting below. Changing this setting from left to hidden will remove the cell toolbar and in turn condense the space between cells. If you're ok with removing the actions in the cell toolbar and cell overflow menu this would give you what you're looking for I believe 

![image](https://user-images.githubusercontent.com/19276747/124342597-d0784200-db79-11eb-988d-590eaf91d7f4.png)

**Comment by ArgoHA** [comment]
@claudiaregio Thank you, that's what I needed
Should I close this issue?

**Comment by LuckyKang79** [comment]
This issue is also reproduced on the latest Release version of VS Code (1.69)

VS Code version: 1.69.0
OS (Windows | Mac | Linux distro) and version: Mac OS Big Sur/Monterey 12.4

![image](https://user-images.githubusercontent.com/93763301/182279666-b9deca0e-3e0b-4e9e-a8b7-1f6addade499.png)

**Comment by LuckyKang79** [comment]
@IanMatthewHuff This issue is also reproduced on the recently Release version of VS Code (1.69/1.70) 

VS Code version: 1.69.0/1.70.2
OS (Windows | Mac | Linux distro) and version: Mac OS Big Sur/Monterey 12.4

**Note**：I am changed this cell toolbar setting from left to hidden as below picture according to the warkaround given by the comment of this issue, but this issue still reproduced. Then, I want to confirm that this issue will be fixed in the future?
![image](https://user-images.githubusercontent.com/93763301/187156236-31b6fb07-97e3-4b58-ad66-e0b3cf5d8e1f.png)

**Comment by IanMatthewHuff** [comment]
@LuckyKang79 I've closed this issue as it's based off of the initial report from @ArgoHA. Your issue doesn't look like the same thing. Could you please open up a new issue on this? It will be easier for us to investigate as an independent issue without the old bug information in this issue.

**Comment by LuckyKang79** [comment]
@IanMatthewHuff Thanks for the details info, I opened a new issue(https://github.com/microsoft/vscode/issues/159522) to track it.
